sdk/highlight-node/src/client.ts

```diff
@@ -105,6 +136,21 @@ export class Highlight {
 		this._log(`Initialized SDK for project ${this._projectID}`)
 	}
 
+	get activeSpanProcessor(): CustomSpanProcessor {
+		// @ts-ignore
+		return trace.getTracerProvider()?._delegate.activeSpanProcessor
```

Comment:
It turns out that the active tracer provider is only available from the `trace` object. `this.processor` isn't actually sending the traces.

Reply:
:/ interesting, feels fragile but i guess the otel js sdk is experimental after all